recipe/build.sh

```diff
@@ -37,7 +37,7 @@ elif [[ "$target_platform" == osx-* ]]; then
   # This is patched by the linker packaged in conda, but when the environment is
   # not activated, clang tries to use the system linker.
   # Adding -B $PREFIX/libexec/pocl makes clang look there for the linker first.
-  EXTRA_HOST_LD_FLAGS="$EXTRA_HOST_LD_FLAGS -undefined dynamic_lookup -B $PREFIX/libexec/pocl"
+  EXTRA_HOST_LD_FLAGS="$EXTRA_HOST_LD_FLAGS -undefined dynamic_lookup -B $PREFIX/libexec/pocl -L/Library/Developer/CommandLineTools/SDKs/MacOSX.sdk/usr/lib -lSystem"
```

Comment:
Can you try
```suggestion
  EXTRA_HOST_LD_FLAGS="$EXTRA_HOST_LD_FLAGS -undefined dynamic_lookup -B $PREFIX/libexec/pocl -lSystem"
```

Reply:
This doesn't work unfortunately:

```
[...]
       Start 123: regression/early_return_before_a_barrier_region_repl
210/268 Test #123: regression/early_return_before_a_barrier_region_repl ..................................***Failed  Error regular expression found in output. Regex=[FAIL]  0.39 sec
CMake Error at $SRC_DIR/cmake/run_test.cmake:34 (message):
  FAIL: Test exited with nonzero code (Subprocess aborted):
  $SRC_DIR/build/tests/regression/test_early_return

  STDOUT:



  STDERR:

  ld: library not found for -lSystem

  error: linker command failed with exit code 1 (use -v to see invocation)

  Final linking of kernel test_kernel failed.
[...]
```

Reply:
Can you try with Line 27 removed?

Reply:
> Can you try with Line 27 removed?

The we wouldn't need `-lSystem` either, right?

Reply:
yep

Reply:
Interesting, with line 27 commented out, and `-lSystem` removed from line 40, it still fails in the same way as indicated above, i.e.:

```

  STDERR:

  ld: library not found for -lSystem

  error: linker command failed with exit code 1 (use -v to see invocation)

  Final linking of kernel test_kernel failed.
```